HW1/P4/P4_bfs.py

```diff
@@ -0,0 +1,36 @@
+def extraArgs(neighbors,distance):
+	"""If a node is in neighbors, and it hasn't been visited, return it. Else return"""
+	def updateNode(n):
```

Comment:
Maybe I'm just not familiar with this style, but what is the purpose of defining this function inside the other?